Sources/SentryCrash/Recording/SentryCrashBinaryImageCache.c

```diff
@@ -0,0 +1,115 @@
+#include "SentryCrashBinaryImageCache.h"
+#include "SentryCrashDynamicLinker.h"
+#include <mach-o/dyld.h>
+#include <pthread.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <unistd.h>
+
+typedef struct SentryCrashBinaryImageNode {
+    SentryCrashBinaryImage image;
+    bool available;
```

Comment:
`l`: I would rename this to `removed` to stick to one naming pattern. The method flipping this flag is called `binaryImageRemoved`. I know I think I suggested `available`, but when looking at it now, I think `removed` is better.

Reply:
If is not a problem for you, I prefer to make the initialization of this struct `{0}` to be in a "non ready" state, so I know I need to activate it in order to make it available.

Reply:
I agree with @brustolin 's approach. In any case, this is set not only from `binaryImageRemoved` but also `binaryImageAdded`, so the name can't really align with one of the other. I think the name is a correct description of what it's representing.